core/ui/AdvancedSearch/Filter.tid

```diff
@@ -79,9 +43,9 @@ caption: {{$:/language/Search/Filter/Caption}}
 				focus={{$:/config/Search/AutoFocus}}
 				configTiddlerFilter="[[$:/temp/advancedsearch]]"
 				firstSearchFilterField="text"
-				inputAcceptActions=<<input-accept-actions>>
-				inputAcceptVariantActions=<<input-accept-variant-actions>>
-				inputCancelActions=<<cancel-search-actions>>
+				inputAcceptActions="<$transclude $variable='search-navigate-actions' targetTiddler={{{ [<tiddler>get[text]] }}}/>"
+				inputAcceptVariantActions="<$transclude $variable='search-navigate-variant-actions' targetTiddler={{{ [<tiddler>get[text]] }}}/>"
+				inputCancelActions="<$transclude $variable='search-cancel-actions' searchTiddler='$:/temp/advancedsearch' inputTiddler='$:/temp/advancedsearch/input' refreshTiddler='$:/temp/advancedsearch/refresh'/>"
```

Comment:
I suggest we use triple double quotes and use double quotes inside. It is much more visible.

```
				inputAcceptActions="""<$transclude $variable="search-navigate-actions" targetTiddler={{{ [<tiddler>get[text]] }}}/>"""
				inputAcceptVariantActions="""<$transclude $variable="search-navigate-variant-actions" targetTiddler={{{ [<tiddler>get[text]] }}}/>"""
				inputCancelActions="""<$transclude $variable="search-cancel-actions" searchTiddler="$:/temp/advancedsearch" inputTiddler="$:/temp/advancedsearch/input" refreshTiddler="$:/temp/advancedsearch/refresh"/>"""
```